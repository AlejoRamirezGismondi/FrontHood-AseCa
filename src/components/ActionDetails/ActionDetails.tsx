import React from 'react';

const ActionDetails = () => {
  return (
    <div>
      <h1 data-testid={"detail-id"}>Hello World</h1>
    </div>
  );
}

export default ActionDetails;
