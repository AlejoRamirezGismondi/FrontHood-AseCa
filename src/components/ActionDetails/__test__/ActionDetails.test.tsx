import React from 'react';
import ReactDOM from 'react-dom';
import ActionDetails from "../ActionDetails";
import {cleanup, render} from "@testing-library/react";
import renderer from 'react-test-renderer';

afterEach(cleanup);

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<ActionDetails/>, div);
})

it('should render correctly', function () {
  const { getByTestId } = render(<ActionDetails/>);
  expect(getByTestId("detail-id")).toHaveTextContent("Hello World")
});

it('should match snapshot', function () {
  const tree = renderer.create(<ActionDetails/>).toJSON();
  expect(tree).toMatchSnapshot();
});
