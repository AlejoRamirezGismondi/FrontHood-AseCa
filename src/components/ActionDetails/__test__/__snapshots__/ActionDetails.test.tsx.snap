// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<div>
  <h1
    data-testid="detail-id"
  >
    Hello World
  </h1>
</div>
`;
